entries/on.xml

```diff
@@ -265,13 +265,21 @@ $( "body" ).on( "click", "p", function() {
 ]]></code>
   </example>
   <example>
-    <desc>Cancel a link's default action using the <code>.preventDefault()</code> method.</desc>
+    <desc>Cancel a link's default action using the <code>.preventDefault()</code> method:</desc>
     <code><![CDATA[
 $( "body" ).on( "click", "a", function( event ) {
   event.preventDefault();
 });
 ]]></code>
   </example>
+  <example>
+    <desc>Multiple events:</desc>
+    <code><![CDATA[
+$( "body" ).on( "click mouseleave", function( event ) {
```

Comment:
Can you write a more real-world example so that other people can benefit from it?